articles/libraries/machine-learning/design.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Quantum machine learning library
+description: Learn the basic concepts of designing circuit models for the quantum circuit centric classifier.
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/17/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.design
+---
+
+# Design your own classifier
+
+In this tutorial you will learn the basic concepts behind the design of circuit models for the quantum circuit centric classifier.
+
+As in classical deep learning, there is no general rule for choosing a specific architecture. Depending on the problem some architectures will perform better than others. But, there are some concepts that might be useful when designing the circuit:
+
+- A large number of parameters implies a more flexible model, which may be suitable to draw complicated classification boundaries but which may also be more susceptible to overfitting.
+
+- Entangling gates between qubits are essential to capture the correlations between the quantum features.
+
+## How to build a classifier with Q\#
+
+To build a classifier we are going to concatenate parametrized controlled rotations in our circuit model. To do it we can use the type [`ControlledRotation`](xref:microsoft.quantum.machinelearning.controlledrotation) defined in the Quantum Machine Learning library. This type accepts four arguments that determine: the index of the target qubit, the array of indices of the control qubits, the axis of rotation, and index of the associated parameter in the array of parameters defining the model.
+
+Let's see an example of a classifier. In the [half-moons sample](link to the sample), we can find the following classifier defined in the file `Training.qs`.
+
+    ```qsharp
```

Comment:
Code isn't rendering properly

Reply:
Looks like it's indented too far, perhaps?